asset_dashboard/models.py

```diff
@@ -195,6 +195,9 @@ def save_zone_distribution(sender, instance, **kwargs):
     def calculate_zone_proportion(cls, distributions_by_zone, total_distribution_area):
         proportions_by_zone = {}
 
+        if total_distribution_area == 0.0:
+            return distributions_by_zone
```

Comment:
#174 was due to a backend failure. the code would delete the asset and then try to divide by 0, aka `ZeroDivisionError: division by zero`